migrate-atmos-s3-to-ecs-s3.html.md.erb

```diff
@@ -0,0 +1,121 @@
+
+*Important*: Please make sure that you have the latest [cf CLI](https://github.com/cloudfoundry/cli) installed before you follow this migration guide.
+
+First, create a new dynstrg-2 service in the same space, let's call it "new-holiday-pictures":
+
+<pre class="terminal">
+$ cf create-service dynstrg-2 usage new-holiday-pictures
+Creating service instance new-holiday-pictures in org INI-DOS-DPS-TCL / space playground
+</pre>
+
+Next, stop all your apps which use dynstrg so that no new objects are stored during the migration:
+
+<pre class="terminal">
+$ cf stop my-awesome-app
+</pre>
+
+As a next step, we'll have to download the minio S3 client. We are going to use it to copy one bucket after another recursively from the old service to the new one.
+
+For all operating systems, visit the minio website [minio](https://www.minio.io/downloads.html#download-client)
+
+To enable access to both services for our minio client, we get the credentials by generating a service key on each service:
+<pre class="terminal">
+cf create-service-key holiday-pictures mys3servicekey
+cf create-service-key new-holiday-pictures mys3servicekey
+</pre>
+
+This will get us the credentials:
+<pre class="terminal">
+cf service-key holiday-pictures mys3servicekey
+{
+ "accessHost": "ds11s3.swisscom.com",
+ "accessKey": "5484335407854a4c9dc88e01206fc148/CF_P8_7AAA80F2_98A1_46F2_9414_A54633972566",
+ "sharedSecret": "7PwzDzLbXOWZiYBoM22C3kp7XgM="
+}
+</pre>
+
+Open the minio config (.mc/config-json) and enter the credentials in the following way:
+
+<strong>*Important*: Please make sure that you use https and set the right api version!</strong>
+<pre class="terminal">
+{
+        "version": "9",
+        "hosts": {
+                "holiday": {
+                        "url": "https://ds11s3.swisscom.com",
+                        "accessKey": "5484335407854a4c9dc88e0b206fc148/CF_P8_7AAA80F2_98A1_46F2_9414_A54633972566",
+                        "secretKey": "7PwzDzLbXOWZiYBoM22C3kp7XgM=",
+                        "api": "S3v2",
+                        "lookup": "auto"
+                },
+                "holiday-dynstrg2": {
+                        "url": "https://ds11s3.swisscom.com",
+                        "accessKey": "85d9b397330add64920a1409f4732119-6683",
+                        "secretKey": "NXuYGhJnilomgZ9PpMbajTD85sn8s+twCBL5KvDq",
+                        "api": "S3v4",
+                        "lookup": "auto"
+                }
+}
+</pre>
+
+Before we can use the mc command (minio) to mirror a bucket, we need to create an empty bucket on the new service. In this example we have two buckets in our old service, "pictures" and "tickets":
+<pre class="terminal">
+mc mb holiday-dynstrg2/tickets
+Bucket created successfully `holiday-dynstrg2/tickets`.
+mc mb holiday-dynstrg2/pictures
+Bucket created successfully `holiday-dynstrg2/pictures`.
+</pre>
+ 
+Now we are finally able to migrate the whole buckets from the old to the new service:
```

Comment:
Finally, you are able to migrate all the buckets from the old to the new service: